gemfirexd/tools/src/test/java/com/pivotal/gemfirexd/jdbc/JdbcTestBase.java

```diff
@@ -76,6 +76,7 @@ protected void setUp() throws Exception {
     //System.setProperty("gemfirexd.TEST_FLAG_OFFHEAP_ENABLE","true");
     GemFireXDUtils.IS_TEST_MODE = true;
     System.setProperty(HDFSStoreImpl.ALLOW_STANDALONE_HDFS_FILESYSTEM_PROP, "true");
+    System.setProperty("gemfire.DISALLOW_RESERVE_SPACE", "true");
```

Comment:
Better to set in TestUtil.doCommonSetup which is invoked by all tests because some don't extend JdbcTestBase.

Reply:
Ok. WIll do. Thanks.